src/app/team/locationForm/number/LocationNumberView.jsx

```diff
@@ -0,0 +1,154 @@
+
+import Button from '../../../../components/button';
+import ConfirmationOptions from '../../../../components/form/ConfirmationOptions';
+import ConfirmationModal from './ConfirmationModal';
+
+import deleteIcon from './DeleteIcon.svg';
+
+const formatPhoneNumber = phone =>
+  `${phone.number.replace(/\./g, '-')}${
+    phone.extension ? ` ext. ${phone.extension}` : ''
+  }`;
+
+class LocationNumberView extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isEditing: false,
+      showDeleteModal: false,
+    };
+
+    this.toggleEdit = this.toggleEdit.bind(this);
+    this.onConfirmDelete = this.onConfirmDelete.bind(this);
+    this.onCancelDelete = this.onCancelDelete.bind(this);
+    this.deletePhone = this.deletePhone.bind(this);
+  }
+
+  async onConfirmDelete() {
+    const { showDeleteModal: { id } } = this.state;
+
+    await this.props.deletePhone(id);
+
+    this.setState({
+      showDeleteModal: false,
+    }, () => {
+      this.props.fetchResourceData();
+    });
+  }
+
+  onCancelDelete() {
+    this.setState({
+      showDeleteModal: false,
+    });
+  }
+
+  deletePhone(phone) {
+    this.setState({
+      showDeleteModal: phone,
+    });
+  }
+
+  toggleEdit() {
+    this.setState({
+      isEditing: !this.state.isEditing,
+    });
+  }
+
+  render() {
+    const { phones, onConfirm, match } = this.props;
+    const { isEditing, showDeleteModal } = this.state;
+
+    return (
+      <div className="w-100">
+        <div style={{ fontSize: '13px', marginBottom: '1em' }} className="font-weight-bold mt-2">
+          Phone Numbers
+        </div>
+
+        { showDeleteModal &&
+          <ConfirmationModal
+            number={formatPhoneNumber(showDeleteModal)}
+            onCancel={this.onCancelDelete}
+            onConfirm={this.onConfirmDelete}
+          /> }
+
+        <ul className="PhoneNumbers-List">
+          {
+            phones.map(phone => (
+              <li key={`selector-${phone.id}`} className="PhoneNumbers-Element">
+                { isEditing &&
+                  <button
+                    onClick={() => this.deletePhone(phone)}
+                    className="PhoneNumber-Delete"
+                  >
+                    <img src={deleteIcon} alt="Delete" />
+                  </button>
+                }
+                <p className="PhoneNumber-Number">{ formatPhoneNumber(phone) }</p>
+                <p className="PhoneNumber-Type">{phone.type}</p>
+
+                { isEditing &&
+                  <Link
+                    to={`${match.url}/${phone.id}`}
+                    className="Button Button-secondary Button-fluid"
+                  >
+                    Edit
+                  </Link>
+                }
+              </li>
+            ))
+          }
+        </ul>
+
+        <Link
+          to={`${match.url}/new`}
+          className="Button Button-secondary Button-fluid PhoneNumbers-Add"
```

Comment:
Since this is now an actual Button component, you shouldn't need to specify the Button classes anymore,
Instead, to avoid making assumptions about implementation details of the component, it can just be:
```
<Button
  onClick={() => { this.props.onNew(); }}
  secondary
  fluid
  className="PhoneNumbers-Add"
>
```

Reply:
Also, while this place has been fixed, a few lines up there's still a Link that uses Button classes. Can that just be a Button too?